source/gameshared/gs_pmove.c

```diff
@@ -42,9 +47,9 @@ int playerbox_gib_viewheight = 8;
 
 #define SPEEDKEY    500
 
-#define PM_DASHJUMP_TIMEDELAY 1000 // delay in milliseconds
-#define PM_WALLJUMP_TIMEDELAY	1300
-#define PM_WALLJUMP_FAILED_TIMEDELAY	700
+#define PM_DASHJUMP_TIMEDELAY (g_dashjump_cooldown->latched_string ? atoi(g_dashjump_cooldown->latched_string) : g_dashjump_cooldown->integer)
```

Comment:
i don't think you should put logic in macros like this, maybe use a global instead?